### PR TITLE
Krishna dhakad intro

### PR DESCRIPTION
**Pull request by Krishna2864**
updated the readme file added intro skills and projects

### CODE REVIEW

File: 2023/Krishna-Dhakad/Readme.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+- Multithreading , multiprocessing 
+
+- Javascript , React 
+
+
+
+- Project i want to contribute 
+  I want to contribute to the project idea `RFC 72: Background workers` , 
+   - here we can use task queueing with Celery ,and message broker with Redis or RebbitmQ
```
Comment (thibaudcolas) [inline 2023/Krishna-Dhakad/Readme.md:42]: RabbitMQ?

File: 2023/Krishna-Dhakad/Readme.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+## **Introduction**
+
+
+
+- Hi , Everyone
+  My name is krishnkant, currently i'm pursuing my btech education in computer enginering from prestige institute of Enginering and 
+  Reaserach Indore, India , im also work as a django devloper at Godlen Eagle Tech ,indore
```
Comment (thibaudcolas) [inline 2023/Krishna-Dhakad/Readme.md:7]: There are quite a few typos here. It’s not the biggest deal but do make sure to check your work with a spellchecker to look more professional.

**Review by thibaudcolas — APPROVED** [review]
Nice one! Markdown formatting is looking a bit odd, I’d suggest revisiting this if you have the chance (and check for typos).

File: 2023/Krishna-Dhakad/Readme.md (hunk 2 above)
Comment (Krishna2864) [reply 2023/Krishna-Dhakad/Readme.md:7]: okay ill take care for that

**Pull request merged**